lib/src/client.dart

```diff
@@ -28,10 +27,7 @@ abstract class Client {
   /// Currently this will create an [IOClient] if `dart:io` is available and
   /// throw an [UnsupportedError] otherwise. In the future, it will create a
   /// [BrowserClient] if `dart:html` is available.
```

Comment:
It can now be importable while also being unavailable in the sense that its API just throws `UnsupportedError`s.